R/boxes.R

```diff
@@ -83,7 +87,12 @@ infoBox <- function(title, value = NULL, subtitle = NULL,
     div(class = "info-box-content",
       span(class = "info-box-text", title),
       if (!is.null(value)) span(class = "info-box-number", value),
-      if (!is.null(subtitle)) p(subtitle)
+      # if (progressBar) div(class = "progress", div(class = "progress-bar", style = progressBarValue)),
+      # if (progressBar) div(class = "progress", div(class = "progress-bar", style =
+      #                                                paste0("width: ", progressBarValue, "%; height: 2px;"))),
+
+      if (progressBar) div(class = "progress", progressBarValue),
+      if (!is.null(subtitle)) span(class = "progress-description", subtitle)
```

Comment:
`span(class = "progress-description", subtitle)` now uses CSS from AdminLTE which makes sure it behave correctly due to margins.